src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -107,6 +110,51 @@ public async Task<IActionResult> GetHostStatus([FromServices] IScriptHostManager
             return Ok(status);
         }
 
+        [HttpGet]
+        [Route("admin/host/workerProcesses")]
```

Comment:
Similar to the Kudu API that exists /api/processes [here](https://github.com/projectkudu/kudu/blob/1df1ce7dad1d941b2658ed0a66f9d09ee6f96a8f/Kudu.Services.Web/App_Start/NinjectServices.cs#L538).

Wondering if our API shouldn't be admin/host/processes and also include the host process?

Reply:
This API is mainly there to assist with remote debugging for dotnet isolated workers. Not sure if adding the host process itself would be of use.

Reply:
That's the current use case, but since we're adding a new API my vote would be to make it general and return host process as well, so in the future if we decide we need to get host process also we don't have to add yet another separate API for that.

Reply:
That makes sense. Let me try and include the host process information as well. Do you know we have record that anywhere?